carservice/views.py

```diff
@@ -83,6 +84,51 @@ def post(self, request):
             return redirect('/car/read/')
         return render(request, 'car_delete.html', {'form': form})
 
+@login_required
+def carsearch(request):
+    search = request.GET.get('search')
+    cars = Car.objects.none()
+    offers = Offer.objects.none()
+    message = ''
+
+    if search:
+        search_terms = search.split()
+        car_brand = ''
+        car_model = ''
+
+        if len(search_terms) > 0:
+            car_brand = search_terms[0]
+            if len(search_terms) > 1:
+                car_model = search_terms[1]
+
+        if car_brand and car_model:
+            cars = Car.objects.filter(Q(car_brand__iexact=car_brand) & Q(car_model__iexact=car_model))
```

Comment:
nie lepiej użyć icontains? Rzadko szukajka działa tak, że tylko konkretna fraza ma się nam wyświetlić

Reply:
jak dawałem icontains to wyszukiwało mi po wpisaniu literki randomowe auta